File: YouTubeSubCount.py
import requests
from bs4 import BeautifulSoup
import re

# Retrieve the web page
url = 'https://www.youtube.com/c/PokerFaceYoutube/about'

# Parse the HTML
req = requests.get(url)
soup = BeautifulSoup(req.text, 'html.parser')
textSoup = str(soup)

regex = '(?<="subscriberCountText":{"accessibility":{"accessibilityData":{"label":").+?(?="}},"simpleText":")'
m = re.search(regex, textSoup)
print(m[0])

File: YouTubeSubCount.js
import axios from 'axios';

const getSubCount = async () => {
    const URL = 'https://corsanywhere.herokuapp.com/https://www.youtube.com/c/PokerFaceYoutube/about';
    try {
        const response = await axios.get (URL);
        const text = response.data;
        // (?<="subscriberCountText":{"accessibility":{"accessibilityData":{"label":")
        // (this is what we want = 3.26k subscribers) = (.+?)
        // (?="}},"simpleText":")
        const regex = '(?<="subscriberCountText":{"accessibility":{"accessibilityData":{"label":")(.+?)(?="}},"simpleText":")';
        const result = text.match(regex)[0]; // Get the first result        
    }
    catch (error) {
        console.log(error);
    }
}
